salt/_modules/metalk8s.py

```diff
@@ -225,8 +225,15 @@ def get_products(products=None):
             continue
 
         info.update({'path': path, 'iso': iso})
-        res.update(
-            {'metalk8s-{0}'.format(version): info}
-        )
+        env_name = 'metalk8s-{0}'.format(version)
```

Comment:
What's the relation with this PR?

Reply:
Not related directly but it's already in a separate commit

Reply:
yes, and it's a good point.

Still, this seems to fix a bug ("to not block boot in case an iso no longer here") so I think it deserves its own issues/PR.

Reply:
It's already refer to the issue #924

Reply:
Ok, good enough then